nodeclient/core_models.go

```diff
@@ -41,40 +40,56 @@ type (
 	InfoResponse struct {
 		// The name of the node software.
 		Name string `json:"name"`
-		// The semver version of the node software.
+		// The server version of the node software.
```

Comment:
This should stay `semver`, for `semantic versioning`. 

```suggestion
		// The semver version of the node software.
```